### PR TITLE
Added cleanup and changes for UI to show in Android

### PR DESCRIPTION
**Pull request by Massimo37**
GitHub Issue: #5

## PR Type
- Bugfix

## Description
Added cleanup and changes for UI to show in Android

## PR Checklist 
- [X] Tested UWP
- [X] Tested iOS
- [X] Tested Android
- [X] Tested WASM
- [X] Contains **No** breaking changes
  > If the pull request contains breaking changes, commit message must contain a detailed description of the action to take for the consumer of this library. As explained by the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)

### CODE REVIEW

File: src/library/Uno.Material/Styles/Controls/BottomNavigationBarItem.xaml [hunk 1]
```diff
@@ -71,6 +71,18 @@
 										<Setter Target="Label.Foreground"
 												Value="{StaticResource BottomNavLabelFocusColorBrush}" />
 									</VisualState.Setters>
+
+									<!-- TODO: Investigate why ColorAnimation isn't working -->
```
Comment (MatFillion) [inline src/library/Uno.Material/Styles/Controls/BottomNavigationBarItem.xaml:75]: Open github issue please!

File: src/samples/Uno.Material.Samples/Uno.Material.Samples.Shared/Content/Controls/BottomNavigationBarSamplePage.xaml [hunk 2]
```diff
@@ -4,26 +4,18 @@
...
 		<TextBlock Text="{x:Bind BottomNavBar.SelectedItem.Label, Mode=OneWay}"
 				   FontWeight="Bold"
 				   HorizontalAlignment="Center"
 				   VerticalAlignment="Center" />
 
-		<controls:BottomNavigationBar x:Name="BottomNavBar"
-									  Grid.Row="1">
+		<controls:BottomNavigationBar x:Name="BottomNavBar">
 			<controls:BottomNavigationBar.Items>
-				<controls:BottomNavigationBarItem Label="Favorites">
```
Comment (MatFillion) [inline src/samples/Uno.Material.Samples/Uno.Material.Samples.Shared/Content/Controls/BottomNavigationBarSamplePage.xaml:26]: some have a label and some not?

File: src/library/Uno.Material/Styles/Application/Colors.xaml [hunk 3]
```diff
@@ -29,22 +29,15 @@
...
-			<StaticResource x:Key="BottomNavNotifBackgroundColorBrush"
-							ResourceKey="ErrorBrush" />
-			<StaticResource x:Key="BottomNavNotifLabelColorBrush"
-							ResourceKey="OnErrorBrush" />
+			<Color x:Key="BottomNavLabelFocusColor">#FFFFFF</Color>
+			<Color x:Key="BottomNavIconFocusColor">#FFFFFF</Color>
+			<Color x:Key="BottomNavLabelColor">#FFFFFF</Color>
+			<Color x:Key="BottomNavIconColor">#FFFFFF</Color>
+			<Color x:Key="BottomNavNotifBackgroundColor">#F85977</Color>
+			<Color x:Key="BottomNavNotifLabelColor">#000000</Color>
```
Comment (MatFillion) [inline src/library/Uno.Material/Styles/Application/Colors.xaml:40]: We can close this PR like this temporily but here's what I think would work better (and avoid duplicating hex values)

```
			<SolidColorBrush x:Key="BottomNavBackgroundColorBrush"
							 Color="{ThemeResource PrimaryColor}"/>
			
			<SolidColorBrush x:Key="BottomNavLabelFocusColorBrush"
							 Color="{ThemeResource OnPrimaryColor}"/>

			<SolidColorBrush x:Key="BottomNavIconFocusColorBrush"
							 Color="{ThemeResource OnPrimaryColor}"/>

			<SolidColorBrush x:Key="BottomNavIconFocusColorBrush"
							 Color="{ThemeResource PrimaryColor}"/>
			
			<SolidColorBrush x:Key="BottomNavBackgroundColorBrush"
							 Color="{ThemeResource PrimaryColor}"/>
			<SolidColorBrush x:Key="BottomNavBackgroundColorBrush"
							 Color="{ThemeResource PrimaryColor}"/>
```

File: src/samples/Uno.Material.Samples/Uno.Material.Samples.Shared/Content/Controls/BottomNavigationBarSamplePage.xaml (hunk 2 above)
Comment (Massimo37) [reply src/samples/Uno.Material.Samples/Uno.Material.Samples.Shared/Content/Controls/BottomNavigationBarSamplePage.xaml:26]: Yes, to demonstrate what it looks like to match Zeplin visuals.

File: src/library/Uno.Material/Styles/Controls/BottomNavigationBarItem.xaml (hunk 1 above)
Comment (Massimo37) [reply src/library/Uno.Material/Styles/Controls/BottomNavigationBarItem.xaml:75]: I'll make a note of this right now and link to this particular code.

File: src/library/Uno.Material/Styles/Application/Colors.xaml (hunk 3 above)
Comment (Massimo37) [reply src/library/Uno.Material/Styles/Application/Colors.xaml:40]: Yes I completely agree, we need to remove this soon.

**Pull request merged**